lib/active_record/connection_adapters/oracle_enhanced_adapter.rb

```diff
@@ -103,7 +103,7 @@ module ConnectionAdapters #:nodoc:
     # Ruby 1.8/1.9 ruby-oci8 gem (which provides interface to Oracle OCI client)
     # or with JRuby and Oracle JDBC driver.
     #
-    # It should work with Oracle 9i, 10g and 11g databases.
+    # It should work with Oracle 9i, 10g, 11g and 12c databases.
     # Limited set of functionality should work on Oracle 8i as well but several features
     # rely on newer functionality in Oracle database.
```

Comment:
@yahonda If Oracle enhanced adapter already does not support Oracle 8i and 9i, I think I will remove it from here. What do you think?